### PR TITLE
samples: bluetooth: hr_coded: Redesign samples

### PR DESCRIPTION
**Pull request by KAGA164**
Redesign the Peripheral HR coded and the Central
HR coded samples to make them consistent with
other Bluetooth samples.

### CODE REVIEW

**Review by alwa-nordic — APPROVED** [review]

**Comment by peknis** [comment]
Remember the changelog.

File: samples/bluetooth/peripheral_hr_coded/README.rst [hunk 1]
```diff
@@ -56,6 +67,10 @@ After programming the sample to your development kit, you can test it by connect
 Dependencies
 ************
 
+This sample uses the following |NCS| libraries:
```
Comment (peknis) [inline samples/bluetooth/peripheral_hr_coded/README.rst:70]:
```suggestion
This sample uses the following |NCS| library:
```

**Review by peknis — CHANGES_REQUESTED** [review]

File: samples/bluetooth/peripheral_hr_coded/README.rst (hunk 1 above)
Comment (KAGA164) [reply samples/bluetooth/peripheral_hr_coded/README.rst]: Done

**Review by peknis — APPROVED** [review]

**Comment by KAGA164** [comment]
> Remember the changelog.

@peknis Do we need to change changelog? The sample have the same functionality like previously, I just redesign a code and add leds. What do you think?

**Comment by peknis** [comment]
> > Remember the changelog.
> 
> @peknis Do we need to change changelog? The sample has the same functionality like previously, I just redesign a code and add leds. What do you think?

I believe we are ok without. Please, remove the changelog-entry-required tag.

**Review by dchat-nordic — APPROVED** [review]

**Pull request merged**